tests/components/homeassistant/test_scene.py

```diff
@@ -117,3 +127,50 @@
     assert scene.name == "hallo_2"
     assert scene.state == "scening"
     assert scene.attributes.get("entity_id") == ["light.kitchen"]
+
+
+async def test_snapshot_service(hass, caplog):
```

Comment:
Let's activate a created scene to make sure that the states were imported correctly ?

Reply:
Done